### ISSUE TITLE
module 'spikeextractors' has no attribute 'AxonaRecordingExtractor'

### ISSUE THREAD

**Issue by lkeegan** [catalystneuro/mease-lab-to-nwb#45]
following the readme installation instructions and then doing `import mease_lab_to_nwb` gives me the following error:

```
/usr/share/miniconda/envs/measelab/lib/python3.8/site-packages/mease_lab_to_nwb/__init__.py:1: in <module>
    from .convert_ced.cednwbconverter import CEDNWBConverter
/usr/share/miniconda/envs/measelab/lib/python3.8/site-packages/mease_lab_to_nwb/convert_ced/cednwbconverter.py:8: in <module>
    from nwb_conversion_tools import NWBConverter, CEDRecordingInterface
/usr/share/miniconda/envs/measelab/lib/python3.8/site-packages/nwb_conversion_tools/__init__.py:3: in <module>
    from .datainterfaces import *
/usr/share/miniconda/envs/measelab/lib/python3.8/site-packages/nwb_conversion_tools/datainterfaces/__init__.py:21: in <module>
    from .ecephys.axona.axonadatainterface import (
/usr/share/miniconda/envs/measelab/lib/python3.8/site-packages/nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py:67: in <module>
    class AxonaRecordingExtractorInterface(BaseRecordingExtractorInterface):
/usr/share/miniconda/envs/measelab/lib/python3.8/site-packages/nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py:70: in AxonaRecordingExtractorInterface
    RX = se.AxonaRecordingExtractor
E   AttributeError: module 'spikeextractors' has no attribute 'AxonaRecordingExtractor'
```

#44 would add some basic CI to setup the conda environment & install mease_lab_to_nwb, which currently also reproduces this error.

**Comment by alejoe91** [comment]
Thank @lkeegan 

We are waiting on the NEO dev team to make a release to also release a new version of SE (which will fix these import issues).
This should happen within the next 2 weeks.

For now, I recommend installing `spikeextractors` and `neo` from the current master branches (sorry about that!)

**Comment by lkeegan** [comment]
resolved by #44
